Patterns/English/English-TimeZone.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 # Direct UTC Expression
 DirectUtcRegex: !simpleRegex
-  def: \b(utc|gmt)(\s*[+\-\u00B1]?\s*[\d]{1,2}(\s*:\s*[\d]{1,2})?)?\b
+  def: \b(utc|gmt|msk)(\s*[+\-\u00B1]?\s*[\d]{1,2}(\s*:\s*[\d]{1,2})?)?\b
```

Comment:
I don't think it's appropriate to mix gmt|utc behaviour with Russian-specific timezones. These are two semantically different concepts.